nbs/tutorials/tutorial.ipynb

```diff
@@ -64,30 +64,32 @@
     "4. nbdev\n",
```

Comment:
<p>Conda has a <code>-y</code> flag</p><p>Looks like pip install jupyterlab will not ask (will only ask for uninstall, in which case you can pass <code>-y</code> there  too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Good to know, will update</p>